canton_dep.bzl

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 canton = {
-    "sha": "cd72960040e3a231f4b922ae8a88bff9a2775ea1df218d1d4b01361acd6e4f54",
```

Comment:
This is a temporary diff. I think the wrong checksum is currently checked into `main`.